src/models/Invite.ts

```diff
@@ -0,0 +1,17 @@
+import { Schema, Document, model, Model } from 'mongoose';
+
+export interface InviteType {
+  key: string;
+  expire: number;
+}
+
+export interface InviteTypeModel extends InviteType, Document {}
+
+export interface InviteDocument extends InviteTypeModel, Document {}
+
+const InviteSchema = new Schema({
+  key: { type: String, required: true },
+  expire: { type: Number, required: true },
+});
+
+export default model<InviteTypeModel>('Invite', InviteSchema);
```

Comment:
스크럼때 컨벤션 Type -> I로 수정했으니 수정해주시면 좋을것 같아요!
```suggestion
export interface IInvite {
  key: string;
  expire: number;
}

export interface IInviteModel extends IInvite, Document {}

export interface IInviteDocument extends IInviteModel, Document {}

const InviteSchema = new Schema({
  key: { type: String, required: true },
  expire: { type: Number, required: true },
});

export default model<IInviteModel>('Invite', InviteSchema);
```